spec/support/schema_helper.rb

```diff
@@ -0,0 +1,39 @@
+InvoiceType = GraphQL::ObjectType.define do
```

Comment:
Just a nitpick, but I would organize this into files/folders just like a typical project, or it will become a junk folder for everybody's spec classes.